sdk/web-pubsub/web-pubsub-client/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "no-return-await": "off",
+    "no-empty": "off",
+    "no-constant-condition": "off"
```

Comment:
any reason to turn them off? I think it's better to fix the errors/warnings, or supress individual ones explicitly.

Reply:
"no-empty": We have some catch blocks that are used to ignore all errors, so where will be some empty blocks
"no-constant-condition": We have a `while(true)` loop, and we break the look in loop. Changing the condition will not meet our requirements.
"no-return-await": In some function we need to return await, this lint seems doesn't make sense